.github/CHANGELOG.md

```diff
@@ -6,12 +6,18 @@
 
 <h3>Bug fixes</h3>
 
+* ``program.compile_info`` and ``program.target`` are correctly set if compilation is successfully
+  completed when compiling a ``TDMProgram``.
```

Comment:
Given that the changelog is written in Markdown, we can replace \`\` with \`. I would also specify that this change only impacts TDM programs compiled using the "TDM" compiler.
```suggestion
* The `TDMProgram.compile_info` and `TDMProgram.target` fields are now set when
  a `TDMProgram` is compiled using the "TDM" compiler.
```